css/sass/dialog.scss

```diff
@@ -15,6 +15,9 @@ $object-field-background: #ebf1f5;
   top: initial;
   max-height: 90vh;
   margin: 5vh auto;
+  .object-tabs {
+    float: left;
+  }
```

Comment:
This causes another layout issue.
![screen shot 2017-12-15 at 4 15 53 am](https://user-images.githubusercontent.com/315275/34042253-8ca6bfd8-e150-11e7-9fa7-3d96c7074899.png)
I think it's better to use clearfix approach to gohan-form-description class.